build.rs

```diff
@@ -1,9 +1,16 @@
 extern crate pkg_config;
+extern crate gcc;
 
-const NAME: &'static str = "sqlite3";
 
+#[cfg(not(feature = "bound"))]
 fn main() {
+    const NAME: &'static str = "sqlite3";
```

Comment:
I moved this line here to prevent a 'unused' warning to be emitted when building with the 'bound' feature.